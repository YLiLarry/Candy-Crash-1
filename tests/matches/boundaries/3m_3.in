3 ..
hint
swap 2 1 right
