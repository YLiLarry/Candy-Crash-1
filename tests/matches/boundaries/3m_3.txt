__0 __1 __2
__0 __3 __2
__3 __2 __1
321
