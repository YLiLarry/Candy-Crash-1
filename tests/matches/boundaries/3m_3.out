
__0 __1 __2 
__0 __3 __2 
__3 __2 __1 
>> hint: 2 1 right

__0 __1 __2 
__0 __3 __2 
__3 __1 __2 

__0 __1 ___ 
__0 __3 ___ 
__3 __1 ___ 

__0 __1 __1 
__0 __3 __2 
__3 __1 __3 
